### PR TITLE
non-production: adopt async, falcon; deprecate EM+Synchrony, goliath

### PR DESCRIPTION
**Pull request by ColinDKelley**

### CODE REVIEW

File: docs/config.json [hunk 1]
```diff
@@ -756,14 +768,20 @@
     },
     {
       "timeline": [
+        {
+          "moved": 1,
+          "ringId": "adopt",
+          "date": "2023-06-05",
+          "description": "Works out-of-the-box with Ruby 3. Actively supported, unlike Synchrony (which is supplants)."
```
Comment (hislopzach) [inline docs/config.json]: ⛏️ Noticed you were fixing typos - when lurking in this PR I found a small one
~(which is supplants)~ => (which it supplants)

Comment (ColinDKelley) [reply docs/config.json]: Thanks for catching that. Fixed: https://github.com/Invoca/tech-radar/pull/7/commits/4a9567eab97c390d1e09b2d4f5313611c612d618

**Comment by ColinDKelley** [comment]
Hey @nburwell Any steps I need to take to "compile" these updates? Anything to do to push these updates into Backstage?

**Review by nburwell — APPROVED** [review]
Looks good, thanks for updating this!
Just merging it will build the updated HTML page (I'm wasn't sure how backstage connects to this to be honest - I was thinking it was an iframe but looks like it does its own rendering from this json file: https://github.com/Invoca/invoca-backstage/blob/b443c96ae030d38f9baf5cb465c2b45959688d83/packages/app/src/lib/TechRadarClient.ts#L6)

So, nothing more to do other than merge!

**Pull request merged**